CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(PROJECT_PATCH_VERSION 11)
 set(PROJECT_VERSION      ${PROJECT_MAJOR_VERSION}.${PROJECT_MINOR_VERSION}.${PROJECT_PATCH_VERSION})
 
 set(CMAKE_C_FLAGS         "${CMAKE_C_FLAGS} -DPROJECT_VERSION=${PROJECT_VERSION} -Wall")
-set(CMAKE_C_FLAGS_DEBUG   "${CMAKE_C_FLAGS_DEBUG} -O0 -ggdb3")
+set(CMAKE_C_FLAGS_DEBUG   "${CMAKE_C_FLAGS_DEBUG} -O0 -ggdb3 -fsanitize=address -fsanitize=leak -fstack-protector-strong")
```

Comment:
We should probably check to make sure these compiler opts are available before setting them. Also, these flags seem like they would be better off in -DCMAKE_BUILD_TYPE=Production.

Thoughts?